pkg/acceptance/bettertestspoc/config/providermodel/gen/main/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	resourcemodelgen "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/bettertestspoc/config/model/gen"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/bettertestspoc/config/providermodel/gen"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/genhelpers"
+)
+
+func main() {
+	genhelpers.NewGenerator(
+		gen.GetProviderSchemaDetails,
+		resourcemodelgen.ModelFromResourceSchemaDetails,
+		getFilename,
+		gen.AllTemplates,
+	).
+		RunAndHandleOsReturn()
+}
+
+func getFilename(_ genhelpers.ResourceSchemaDetails, model resourcemodelgen.ResourceConfigBuilderModel) string {
```

Comment:
Nit: we already have multiple funcs like this, and they could be extracted.

Reply:
they use different attributes, I don't think see the value of extracting these oneliners that are each-generator-specific